src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -43,88 +38,237 @@
+              "minProperties": 1,
+              "patternProperties": {
+                "^.*$": {
+                  "additionalProperties": false,
+                  "properties": {
+                    "longname": {
+                      "type": "string"
+                    },
+                    "profile_type": {
+                      "oneOf": [
+                        {
+                          "additionalProperties": false,
+                          "properties": {
+                            "name": {
+                              "const": "fixed"
+                            },
+                            "surface_value": {
+                              "type": "number"
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "surface_value"
+                          ],
+                          "type": "object"
+                        },
+                        {
+                          "additionalProperties": false,
+                          "properties": {
+                            "name": {
+                              "const": "profile"
+                            },
+                            "surface_value": {
+                              "type": "number"
+                            },
+                            "top_value": {
+                              "type": "number"
+                            }
+                          },
+                          "required": [
+                            "name",
+                            "surface_value",
+                            "top_value"
+                          ],
+                          "type": "object"
+                        }
+                      ]
+                    },
+                    "units": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "longname",
+                    "profile_type",
+                    "units"
+                  ],
+                  "type": "object"
+                }
+              },
+              "type": "object"
+            }
+          },
+          "required": [
+            "base_file"
+          ],
+          "type": "object"
         },
         "length": {
           "minimum": 1,
           "type": "integer"
         },
         "model_configure": {
-          "$ref": "#/$defs/updatable_config"
-        },
-        "mpicmd": {
-          "type": "string"
+          "additionalProperties": false,
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "minProperties": 1,
+              "patternProperties": {
+                "^.*$": {
+                  "type": [
+                    "boolean",
+                    "number",
+                    "string"
+                  ]
```

Comment:
We are going to have problems with `model_configure` as a YAML. I'm almost ready to consider proposing that we update the model code to use a YAML parser for this file. I am not there just yet.

I know that we already have one problem with standard model_configure files -- they use the file type entry with too many quotes like this:

```
filename_base:           'dyn''phy'
```

I think it would be a helpful upgrade to the model to make this "almost YAML" file an actual YAML file. Thoughts?

This is a lot of risk at the moment. I would be fine if we wanted to treat this file as a flat text file template until we can ensure it's going to be YAML compliant.

Reply:
`model_configure` as-is is the worst of both worlds, a YAML/namelist chimera. IMO UFS should pick one or the other and use it. If it is being read by something other than Fortran, probably YAML is the right language, but if there's an easier path to implementation (e.g. a library for a different config language is already in UFS, or is easier to add than a YAML library), probably that would be fine, too.

I think templating would be a safe alternative in the short term, though unfortunate in that the app or user will have to define everything in advance.

Reply:
@christinaholtNOAA What do we want to do with this in the short term? If FV3 _requires_ a `model_configure` file, then we need to require a `model_configure` YAML entry with some combination of `base_file` and/or `update_values`. Should we treat it the same as `field_table`, i.e. at-least-one handling of `base_file` and `update_values` so that the user can 1. Provide only a file, 2. Provide only values, 3. Provide and file and values to update the file with?